test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml

```diff
@@ -18,11 +18,11 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=/var/lib/kubelet/plugins/pd.csi.storage.gke.io/csi.sock"
```

Comment:
Grab the changes from https://github.com/kubernetes/kubernetes/pull/71146 since that will cause merge conflicts when you merge. (It will change `--csi-address` here `plugin-dir` mountPath below.